### ISSUE TITLE
Page-Notice: confirmation color contrast issue

### ISSUE THREAD

**Issue by edvardsniedre** [eBay/skin#1818]
Confirmation page notice color contrast warning reported by `axe` a11y tool. Tested on Skin 14.1.0.

<img width="481" alt="Screenshot 2022-07-26 at 19 14 57" src="https://user-images.githubusercontent.com/7050068/181085724-51b07fc5-0210-4595-8b59-d77db8e0c495.png">

From what I have confirmed Skin 14 should use the same bg color as Skin 13.
<img width="723" alt="Screenshot 2022-07-26 at 19 45 39" src="https://user-images.githubusercontent.com/7050068/181087011-5552ecfe-770b-48e3-834c-9fd5630dd065.png">

**Comment by ianmcburnie** [comment]
Dev note: our alias should be using `--color-green-6` instead of `--color-green-4`.
